### PR TITLE
Feature/filter buttons

### LINKED ISSUE #95: Feed/Filter by Location UI - Clear Filters Button
NOTE: This is ONLY for the Clear Filters portion of the UI - NOT the Location title, the - button, the zip code field, the Share My Location button, or the Apply filters button

* Pill button with light theme
* Upon tapping - closes Location filter view (if open); clears Location selection (if made)
* Localized

Related to: https://github.com/FightPandemics/FightPandemics-iOS/issues/96

Screen Shot 2020-05-17 at 5 34 59 PM

### LINKED ISSUE #96: Feed/Filter by Location UI - Apply Filters Button
NOTE: This is ONLY for the Clear Filters portion of the UI - NOT the Location title, the - button, the zip code field, the Share My Location button, or the Clear Filters button

* Pill button with dark theme
* Inactive if no Filters applied
* Becomes active if any Filters are applied
* Returns to inactive state if all Filters manually removed
* Returns to inactive state if "Clear filters" button is tapped
* If active and tapped, applies Filter selection globally
* Localized

Related to: https://github.com/FightPandemics/FightPandemics-iOS/issues/95

Screen Shot 2020-05-17 at 5 34 59 PM

### PR DESCRIPTION
**Pull request by ethanswift**
**What this PR does**:

Clear filters and Apply filters buttons on location cell. 

### Checklist

- [x] Compiler warnings resolved
- [x] Linter warnings resolved
- [x] User-facing strings in Localizable.strings file
- [x] Unused code, print statements, and comments removed

**Which issue(s) this PR fixes**:

Resolves #

creates two buttons for the filter's location cell in two separate classes.  
Also: 
Pill button with dark theme
Inactive if no Filters applied
Becomes active if any Filters are applied
Returns to inactive state if all Filters manually removed
Returns to inactive state if "Clear filters" button is tapped
If active and tapped, applies Filter selection globally

**Special notes for reviewers**:

**Additional comments**:

### CODE REVIEW

File: FightPandemics/Features/Filters/ApplyFilterBtn.swift [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+//  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+//  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+//  THE SOFTWARE.
+
+import UIKit
+
+class ApplyFilterBtn: UIView {
+    private var applyFilterBtn = UIButton()
+    func setUp(firltersSelected: Bool, clearFiltersTapped: Bool) {
+        if !firltersSelected || clearFiltersTapped {
```
Comment (stavares843) [inline FightPandemics/Features/Filters/ApplyFilterBtn.swift]: typo - filtersSelected

**Review by OnyekachiSamuel — COMMENTED** [review]
Nice one for the efforts @ethanswift, the apply filter button, and clear filter button should actually be a custom button and not a view. I think you actually need to subclass the UIButton class and customize the button appearance according to what we have in the designs respectively.

**Comment by ethanswift** [comment]
> Nice one for the efforts @ethanswift, the apply filter button, and clear filter button should actually be a custom button and not a view. I think you actually need to subclass the UIButton class and customize the button appearance according to what we have in the designs respectively.

I wanted to do so, but I thought maybe UIView's layout be more desirable in this case.

**Comment by ethanswift** [comment]
@OnyekachiSamuel  Anyway, its fixed now

**Comment by ethanswift** [comment]
mistakenly removed @hkellaway from reviewers here, can't add it again. @hkellaway , please take a look (the name doesn't appear in the list)

**Review by OnyekachiSamuel — APPROVED** [review]

**Comment by ethanswift** [comment]
@OnyekachiSamuel regulated, take a look

**Pull request merged**